near-sdk/src/types/event.rs

```diff
@@ -0,0 +1,95 @@
+pub trait StandardEvent {
+    fn format(&self) -> String;
+    fn emit(&self);
+}
+
+pub struct Event {}
+impl Event {
+    pub fn emit(standard_event: impl StandardEvent) {
+        crate::env::log_str(&format!("EVENT_JSON:{}", &standard_event.format()));
+    }
+}
```

Comment:
This should just be removed altogether. It's basically a duplicate of the other trait and would be confusing to have this type. 

Also, side note is you don't need an owned value of `standard_event` here

Reply:
This was added to provide the pattern `Event::emit(event)`. Personally, I like this more than using the `.emit()` method since it is easier to locate when skimming a contract, but we can go the `.emit()` method route if yall like that more. I do agree there should only be one option to emit events.

Reply:
You could still do that pattern using the other trait, though. What's the need to have it as a separate trait?

Reply:
okay, I see what you are saying